helpers/userProfile.db.ts

```diff
@@ -27,7 +27,7 @@ export const userNameExists = async (username: string) => {
 export const userProfileExists = async (walletAddress: string) => {
```

Comment:
rather than doing a bunch of lowercase you can just create a function that returns the filter object
```const getFilterObject = (value) => {
$regex:value,
$options: "i"
}```

Reply:
then you can just call this function anywhere you are looking for documents